doc/manual/installing.rst

```diff
@@ -133,15 +135,15 @@ Installing the host-side software
         $ cd ~/artiq-dev/llvm-or1k
         $ mkdir build
         $ cd ~/artiq-dev/llvm-or1k/build
-        $ ../configure --prefix=/usr/local/llvm-or1k
-        $ make ENABLE_OPTIMIZED=1 -j4
-        $ sudo -E make install ENABLE_OPTIMIZED=1
+        $ cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local/llvm-or1k -DLLVM_TARGETS_TO_BUILD=OR1K -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=ON
```

Comment:
I think the conda package right now takes for granted that llvmlite is linked statically with llvm.
We can use shared library, but we would need to do a conda package for llvm then and ship it as an llvmlite dependency.